code.sh

```diff
@@ -298,7 +298,7 @@ fi
 # different toolboxes step on each other.
 
 homevar="$(flatpak-spawn --host podman inspect "$container_name" \
-    --format='{{ range .Config.Env }}{{ . }}{{"\n"}}{{ end }}' \
+    --format='{{ range .Config.Env }}{{ . }}\n{{ end }}' \
```

Comment:
If I try this with podman-3.4.4 from Fedora 35, I simply get literal "\n" in the output instead of newlines.

Reply:
I tested with 35 and updated pull-request. See how does it now.